### DISCUSSION TITLE
Stuck on the EULA check pass.

### DISCUSSION THREAD

**Discussion by Hcfa8y** [AchetaGames/Epic-Asset-Manager#273] [Q&A]
Hi, thanks for this cute app. Could some one please help with the engine download process. I'm stuck on the 'EULA accepted' step. The 'https://www.unrealengine.com/en-US/eulacheck/unreal' page where I'm redirected to sign it doesn't exist any more. How else the EULA check can be passed, so I could download the engine?

![1](https://github.com/user-attachments/assets/efd7c889-e71c-4943-bd19-7dda9ab712bd)

**Comment by aknarts** [comment]
Yeah this is a known issue download the engine manually here: https://unrealengine.com/linux

Sadly they changed how their cookies are handled and I have not had much luck breaking through that again.

I do have a "fix" ready, it just relies on couple other things to tighten up. (Summer is very hectic for me)

**Reply by Hcfa8y** [reply]
Thanks for your feedback and for your efforts. And wishing you a long sleepy weekend.
